type-check.ts

```diff
@@ -25,10 +25,22 @@ defaultGlobalFunctions.set("min", [[NUM, NUM], NUM]);
 defaultGlobalFunctions.set("pow", [[NUM, NUM], NUM]);
 defaultGlobalFunctions.set("print", [[CLASS("object")], NUM]);
 
+const defaultGlobalClasses = new Map();
+const rangeFields = new Map<string, Type>();
+rangeFields.set("start", {tag: "number"});
+rangeFields.set("stop", {tag: "number"});
+rangeFields.set("current", {tag: "number"});
+
+const rangeMethods = new Map();
+rangeMethods.set("__init__", [{tag: "number"}, {tag: "number"}, {tag: "class"}])
+rangeMethods.set("hasNext", [{tag: "class"}])
+rangeMethods.set("next", [{tag: "class"}])
+defaultGlobalClasses.set("range", [rangeFields, rangeMethods]);
```

Comment:
Is this used anywhere? I am not sure I understand clearly if you are working on tuples or the range class.

Reply:
This has been removed now. Please refer the earlier comment detailing our decision : https://github.com/ucsd-cse231-s22/chocopy-wasm-compiler-B/pull/8#discussion_r877663827